content/en/database_monitoring/setup_oracle/_index.md

```diff
@@ -85,52 +90,66 @@ Set `APIKEY` and run the following command in the command prompt inside the dire
 ```shell
 start /wait msiexec /qn /i datadog-agent-7.46.0-dbm-oracle-beta-0.33-1.x86_64.msi APIKEY="" SITE="datadoghq.com"
 ```
+[4]: https://s3.amazonaws.com/ddagent-windows-stable/beta/datadog-agent-7.46.0-dbm-oracle-beta-0.33-1.x86_64.msi
+[8]: https://ddagent-windows-stable.s3.amazonaws.com/
 
-##### Docker
-
-The docker beta images can be found [here][9].
+{{% /tab %}}
+{{% tab "Docker" %}}
+The Docker beta images can be found [here][9].
 
 Set `DD_API_KEY` and run the following command to install the beta release, for example:
 
 ```shell
 docker run -d --name dd-agent -v /var/run/docker.sock:/var/run/docker.sock:ro -v /proc/:/host/proc/:ro -v /sys/fs/cgroup/:/host/sys/fs/cgroup:ro -e DD_API_KEY="" -e DD_SITE="datadoghq.com" gcr.io/datadoghq/agent:7.46.0-dbm-oracle-beta-0.33
 ```
 
-##### Latest Agent version
+[9]: https://hub.docker.com/r/datadog/agent/tags?page=1&name=oracle
+{{% /tab %}}
+{{< /tabs >}}
 
-The following beta builds contain implemented Oracle DBM features:
-- Linux: `7.46.0~dbm~oracle~beta~0.33-1`
-- Windows: `7.46.0-dbm-oracle-beta-0.33-1`
-- Docker: `7.46.0-dbm-oracle-beta-0.33`
+### Upgrade an existing Agent installation
 
-### Oracle client
+<div class="alert alert-info">For the latest Oracle monitoring features and bug fixes, <a href="/database_monitoring/setup_oracle/?tab=linux#recommended-agent-version">install the recommended beta version</a> instead of upgrading your existing Agent below.</div>
```

Comment:
I feel that this section doesn't capture the scenario where someone may be upgrading from a custom build to a new custom build or from an official version to a custom build

Reply:
could you say more about why you added these parts?
<img width="706" alt="image" src="https://github.com/DataDog/documentation/assets/46852208/574cc2ea-fe89-49f3-9450-5e688eb5dd2e">

Reply:
could you say more about why you added these parts?
<img width="690" alt="image" src="https://github.com/DataDog/documentation/assets/46852208/0cadcdca-cd3a-43cc-bdf3-51c9f678c694">